events/logger/logger.go

```diff
@@ -118,11 +119,16 @@ func dataRequestAPIToString(
 		b.WriteString(fmt.Sprintf("Error: %s\n", data.Error))
 	}
 	if data.Response != nil {
+		tmpBody := ""
```

Comment:
[small] maybe let's set it to `"(binary data)" just to indicate that there is something but it's binary?

Reply:
good idea, yes, I'll add this